### ISSUE TITLE
NoMethodError: undefined method `include?' for #<Proc...>

### ISSUE THREAD

**Issue by manuelmeurer** [gregnavis/active_record_doctor#112]
I just installed active_record_doctor 1.10.0 and ran it for the first time and the following error was thrown:

```
% bundle exec rake active_record_doctor --trace
** Invoke active_record_doctor (first_time)
** Invoke active_record_doctor:setup (first_time)
** Invoke environment (first_time)
** Execute environment
** Execute active_record_doctor:setup
** Execute active_record_doctor
rake aborted!
NoMethodError: undefined method `include?' for #<Proc:0x0000000113da8568 /Users/manuel/code/myapp/app/app/models/admin_users/admin_user.rb:46 (lambda)>

            !validator.options.fetch(:in, []).include?(nil)
                                             ^^^^^^^^^
/Users/manuel/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/active_record_doctor-1.10.0/lib/active_record_doctor/detectors/missing_presence_validation.rb:57:in `block in inclusion_validator_present?'
/Users/manuel/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/active_record_doctor-1.10.0/lib/active_record_doctor/detectors/missing_presence_validation.rb:54:in `any?'
/Users/manuel/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/active_record_doctor-1.10.0/lib/active_record_doctor/detectors/missing_presence_validation.rb:54:in `inclusion_validator_present?'
/Users/manuel/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/active_record_doctor-1.10.0/lib/active_record_doctor/detectors/missing_presence_validation.rb:46:in `validator_present?'
/Users/manuel/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/active_record_doctor-1.10.0/lib/active_record_doctor/detectors/missing_presence_validation.rb:31:in `block (2 levels) in detect'
/Users/manuel/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/active_record_doctor-1.10.0/lib/active_record_doctor/detectors/missing_presence_validation.rb:29:in `each'
/Users/manuel/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/active_record_doctor-1.10.0/lib/active_record_doctor/detectors/missing_presence_validation.rb:29:in `block in detect'
/Users/manuel/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/active_record_doctor-1.10.0/lib/active_record_doctor/detectors/missing_presence_validation.rb:26:in `each'
/Users/manuel/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/active_record_doctor-1.10.0/lib/active_record_doctor/detectors/missing_presence_validation.rb:26:in `detect'
/Users/manuel/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/active_record_doctor-1.10.0/lib/active_record_doctor/detectors/base.rb:50:in `run'
/Users/manuel/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/active_record_doctor-1.10.0/lib/active_record_doctor/detectors/base.rb:17:in `run'
/Users/manuel/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/active_record_doctor-1.10.0/lib/active_record_doctor/runner.rb:15:in `block in run_one'
/Users/manuel/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/active_record_doctor-1.10.0/lib/active_record_doctor/errors.rb:5:in `handle_exception'
/Users/manuel/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/active_record_doctor-1.10.0/lib/active_record_doctor/runner.rb:14:in `run_one'
/Users/manuel/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/active_record_doctor-1.10.0/lib/active_record_doctor/runner.rb:26:in `block in run_all'
/Users/manuel/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/active_record_doctor-1.10.0/lib/active_record_doctor/runner.rb:25:in `each'
/Users/manuel/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/active_record_doctor-1.10.0/lib/active_record_doctor/runner.rb:25:in `run_all'
/Users/manuel/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/active_record_doctor-1.10.0/lib/active_record_doctor/rake/task.rb:60:in `block in define'
/Users/manuel/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rake-13.0.6/lib/rake/task.rb:281:in `block in execute'
/Users/manuel/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rake-13.0.6/lib/rake/task.rb:281:in `each'
/Users/manuel/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rake-13.0.6/lib/rake/task.rb:281:in `execute'
/Users/manuel/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/sentry-ruby-5.4.2/lib/sentry/rake.rb:26:in `execute'
/Users/manuel/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rake-13.0.6/lib/rake/task.rb:219:in `block in invoke_with_call_chain'
/Users/manuel/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rake-13.0.6/lib/rake/task.rb:199:in `synchronize'
/Users/manuel/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rake-13.0.6/lib/rake/task.rb:199:in `invoke_with_call_chain'
/Users/manuel/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rake-13.0.6/lib/rake/task.rb:188:in `invoke'
/Users/manuel/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rake-13.0.6/lib/rake/application.rb:160:in `invoke_task'
/Users/manuel/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rake-13.0.6/lib/rake/application.rb:116:in `block (2 levels) in top_level'
/Users/manuel/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rake-13.0.6/lib/rake/application.rb:116:in `each'
/Users/manuel/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rake-13.0.6/lib/rake/application.rb:116:in `block in top_level'
/Users/manuel/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rake-13.0.6/lib/rake/application.rb:125:in `run_with_threads'
/Users/manuel/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rake-13.0.6/lib/rake/application.rb:110:in `top_level'
/Users/manuel/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rake-13.0.6/lib/rake/application.rb:83:in `block in run'
/Users/manuel/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rake-13.0.6/lib/rake/application.rb:186:in `standard_exception_handling'
/Users/manuel/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rake-13.0.6/lib/rake/application.rb:80:in `run'
/Users/manuel/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/rake-13.0.6/exe/rake:27:in `<top (required)>'
/Users/manuel/.rbenv/versions/3.1.2/bin/rake:25:in `load'
/Users/manuel/.rbenv/versions/3.1.2/bin/rake:25:in `<top (required)>'
/Users/manuel/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bundler-2.3.20/lib/bundler/cli/exec.rb:58:in `load'
/Users/manuel/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bundler-2.3.20/lib/bundler/cli/exec.rb:58:in `kernel_load'
/Users/manuel/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bundler-2.3.20/lib/bundler/cli/exec.rb:23:in `run'
/Users/manuel/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bundler-2.3.20/lib/bundler/cli.rb:485:in `exec'
/Users/manuel/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bundler-2.3.20/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
/Users/manuel/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bundler-2.3.20/lib/bundler/vendor/thor/lib/thor/invocation.rb:127:in `invoke_command'
/Users/manuel/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bundler-2.3.20/lib/bundler/vendor/thor/lib/thor.rb:392:in `dispatch'
/Users/manuel/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bundler-2.3.20/lib/bundler/cli.rb:31:in `dispatch'
/Users/manuel/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bundler-2.3.20/lib/bundler/vendor/thor/lib/thor/base.rb:485:in `start'
/Users/manuel/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bundler-2.3.20/lib/bundler/cli.rb:25:in `start'
/Users/manuel/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bundler-2.3.20/exe/bundle:48:in `block in <top (required)>'
/Users/manuel/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bundler-2.3.20/lib/bundler/friendly_errors.rb:120:in `with_friendly_errors'
/Users/manuel/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bundler-2.3.20/exe/bundle:36:in `<top (required)>'
/Users/manuel/.rbenv/versions/3.1.2/bin/bundle:25:in `load'
/Users/manuel/.rbenv/versions/3.1.2/bin/bundle:25:in `<main>'
Tasks: TOP => active_record_doctor
```

The code at admin_user.rb:46 is:

```
validates :my_option, inclusion: { in:      -> { _1.role?(:superadmin, :admin) ?
                                                 [true, false] :
                                                 [false] },
                                   message: "can only be selected for admins with role superadmin or admin" }
```

Let me know if I can do anything else to debug!

**Comment by fatkodima** [comment]
Thanks for reporting. Should be fixed by #113.

**Comment by gregnavis** [comment]
@manuelmeurer, thank you for reporting. @fatkodima, thanks for the quick fix.

I'm reopening the issue as the fix hasn't been released yet.
